src/views/newConnection/base.tsx

```diff
@@ -15,6 +15,10 @@ export type FormFeedback =
       type: 'error'
       error: string
     }
+  | {
+      type: 'success'
+      message: string
+    }
```

Comment:
How is this distinct from `message`?

Reply:
Not much right now. This will be updated along with some `type` suggestions